s32/mcux/devices/S32Z270/S32Z270_device.h

```diff
@@ -1885,4 +1885,849 @@ typedef struct {
+        __IO uint32_t DSICR0;                            /**< DSI Configuration Register 0, offset: 0xBC */
+        __I  uint32_t SDR0;                              /**< DSI Serialization Data Register 0, offset: 0xC0 */
+        __IO uint32_t ASDR0;                             /**< DSI Alternate Serialization Data Register 0, offset: 0xC4 */
+        __I  uint32_t COMPR0;                            /**< DSI Transmit Comparison Register 0, offset: 0xC8 */
+        __I  uint32_t DDR0;                              /**< DSI Deserialization Data Register 0, offset: 0xCC */
+        __IO uint32_t DSICR1;                            /**< DSI Configuration Register 1, offset: 0xD0 */
+        __IO uint32_t SSR0;                              /**< DSI Serialization Source Select Register 0, offset: 0xD4 */
+        uint8_t RESERVED_2[16];
+        __IO uint32_t DIMR0;                             /**< DSI Deserialized Data Interrupt Mask Register 0, offset: 0xE8 */
+        __IO uint32_t DPIR0;                             /**< DSI Deserialized Data Polarity Interrupt Register 0, offset: 0xEC */
+        __I  uint32_t SDR1;                              /**< DSI Serialization Data Register 1, offset: 0xF0 */
+        __IO uint32_t ASDR1;                             /**< DSI Alternate Serialization Data Register 1, offset: 0xF4 */
+        __I  uint32_t COMPR1;                            /**< DSI Transmit Comparison Register 1, offset: 0xF8 */
+        __I  uint32_t DDR1;                              /**< DSI Deserialization Data Register 1, offset: 0xFC */
+        __IO uint32_t SSR1;                              /**< DSI Serialization Source Select Register 1, offset: 0x100 */
+        uint8_t RESERVED_3[16];
+        __IO uint32_t DIMR1;                             /**< DSI Deserialized Data Interrupt Mask Register 1, offset: 0x114 */
+        __IO uint32_t DPIR1;                             /**< DSI Deserialized Data Polarity Interrupt Register 1, offset: 0x118 */
+        __IO uint32_t CTARE[6];                          /**< Clock and Transfer Attributes Register Extended, array offset: 0x11C, array step: 0x4 */
+        uint8_t RESERVED_4[8];
+        __I  uint32_t SREX;                              /**< Status Register Extended, offset: 0x13C */
+        uint8_t RESERVED_5[16];
+        __IO uint32_t TSL;                               /**< Time Slot Length Register, offset: 0x150 */
+        __IO uint32_t TS_CONF;                           /**< Time Slot Configuration Register, offset: 0x154 */
+} SPI_Type;
+
+
+/* ----------------------------------------------------------------------------
+   -- DSPI Register Masks
+   ---------------------------------------------------------------------------- */
+
+/*!
+ * @addtogroup DSPI_Register_Masks DSPI Register Masks
+ * @{
+ */
+
+/*! @name MCR - Module Configuration Register */
+/*! @{ */
+
+#define SPI_MCR_HALT_MASK                          DSPI_MCR_HALT_MASK

```

Comment:
same comment as before, can we avoid reformatting the file after copied over from mcux-sdk?

Reply:
ignore this